target_postgres/connector.py

```diff
@@ -24,3 +26,29 @@ def create_sqlalchemy_connection(self) -> sqlalchemy.engine.Connection:
             A newly created SQLAlchemy engine object.
         """
         return self.create_sqlalchemy_engine().connect()
+    
+    def create_sqlalchemy_engine(self) -> sqlalchemy.engine.Engine:
+        """Return a new SQLAlchemy engine using the provided config.
+        
+        Returns:
+            A newly created SQLAlchemy engine object.
+        """
+        return sqlalchemy.create_engine(
```

Comment:
Nice I'm not sure about `values_plus_batch` is there a reason you added that here? 

future=True is interesting as well, might be best to just jump to the new versions!

Reply:
I put the `values_plus_batch` there to match the examples SQLAlchemy gave for postgres fast execution helpers.  https://docs.sqlalchemy.org/en/14/dialects/postgresql.html#psycopg2-fast-execution-helpers

The `future=True` is to assist in migration from 1.4 to 2.0.  Here is the link I got that from. https://docs.sqlalchemy.org/en/14/changelog/migration_20.html#migration-to-2-0-step-four-use-the-future-flag-on-engine